### DISCUSSION TITLE
qq交流群

### DISCUSSION THREAD

**Discussion by mikumifa** [mikumifa/biliTickerBuy#353] [公告]
520420620

**Comment by WittF** [comment]
本群主要用于聊天讨论，请先在Github/文档寻找答案~
先看看Wiki有没有教你
https://github.com/mikumifa/biliTickerBuy/wiki
再翻翻Discussions有没有反馈过
https://github.com/mikumifa/biliTickerBuy/discussions
接着找找Issues有没有相关的
https://github.com/mikumifa/biliTickerBuy/issues
如果还没解决，欢迎加群交流

**Reply by WittF** [reply]
再次重申一下，请确定您的问题无法在Github这些地方解决再加群，~~群里大伙不喜欢笨蛋问题哈哈~~

**Reply by PureVoyage** [reply]
群里乱踢人是什么意思？不让随便问问题，然后不活跃的也给人家踢了是吧，你这个群的意义是什么？

**Reply by WittF** [reply]
@PureVoyage 
> 不让随便问问题

1. 我说明的很清楚了诶，并非不让随便问问题，只是如果提的问题过于简单，或是在Github或文档已经说明过的，仍在群里询问就是在浪费大家的时间，而且就算提的问题很小白，我也并不会做什么。就是群内的各位不是客服，真的解决不了的问题欢迎在群内讨论并寻求帮助

> 然后不活跃的也给人家踢了是吧

2. 长期不活跃的我一般都有@一次，在一段时间后才会踢出，或者是进来了好久只发过一两句的，我个人认为这种成员并没什么必要加入用于交流的本群，如果被踢了欢迎重新加群，我都会通过的

喵喵喵

**Reply by MoYuanCN** [reply]
> 群里乱踢人是什么意思？不让随便问问题，然后不活跃的也给人家踢了是吧，你这个群的意义是什么？

你好，进群要提供订单号的，你没有的话只能这样做了喵

**Reply by MUSESTAR-Official** [reply]
> 群里乱踢人是什么意思？不让随便问问题，然后不活跃的也给人家踢了是吧，你这个群的意义是什么？

您好，我是客服6号，小红书分享链接也是可以的喵

**Reply by VerQust** [reply]
> 群里乱踢人是什么意思？不让随便问问题，然后不活跃的也给人家踢了是吧，你这个群的意义是什么？

您好，客服三号为您服务！根据您反馈的问题，我们群是要求提供闲鱼订单号或者小红书账号以及其他证明材料的哦！这也是无奈之举，毕竟黄牛太多了喵😉

**Reply by YiQunSheng** [reply]
没有啥问题，就是看到很多人在卖佬的软件很心痛。 沟槽的黄牛。
